docusaurus.config.js

```diff
@@ -6,6 +6,71 @@ const remarkCodesandbox = require("remark-codesandbox");
 const path = require("path");
 const isProd = process.env.NODE_ENV === 'production';
 
+const DOCS_REPO_SRC = "https://raw.githubusercontent.com/INFURA/docs/main/docs";
+
+const INFURA_DOCS_PAGES = [
+  // {
+  //   name: "network-endpoints",
+  //   token: "GHSAT0AAAAAACFCLQUUDH4GNNQJTIKNUUPCZL2O3SA",
```

Comment:
Not sure what this token is, but is it ok to display publicly?

Reply:
I believe so, as this is the method @TrofimovAnton85 provided to configure remote pages in [this Jira issue](https://infura.atlassian.net/browse/ACT-996). I'm currently having trouble porting pages from subfolders so I'm waiting to hear from Anton.